odoo/models.py

```diff
@@ -1974,9 +1983,16 @@ def clear_caches(cls):
         ``tools.ormcache`` or ``tools.ormcache_multi``.
         """
         cls.pool._clear_cache()
-        tcbk = traceback.extract_stack()
```

Comment:
Also I think we could skip this particular change, as it is reversed in the following commit